kimsoohyun/app.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const logger = require('morgan');
+const cors = require('cors')
+const app = express()
+const dotenv = require("dotenv")
+const { DataSource } = require('typeorm');
+
+dotenv.config()
+
+app.use(cors())
+app.use(logger('combined'));
+
+app.get('/ping', function (req, res, next) {
+    res.json({message: 'pong'})
+})
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+app.listen(8005, function () {
+    console.log('server listening on port 8005')
+})
```

Comment:
1. 서버를 실행하는 함수로 만들어서 함수를 호출하여 listen 시켜주세요!
2. PORT 번호는 하드코딩이 아닌 .env에 PORT라는 변수에 값을 넣어서 app.js에서 불러와 호출할 수 있도록 수정해주세요!
```suggestion
const PORT = process.env.PORT

const startServer = () => {
  app.listen(PORT, function () {
      console.log(`server listening on port ${PORT}`)
  })
}

startServer()
```